Umbraco-StarterKit/Widgets/Grid/Price-list/index.md

```diff
@@ -0,0 +1,22 @@
+# Price List
+
+The Price List, lets you create a list of items with a title, description, and price of the items. 
+The way it works is that you click the add content button next to items, then fill out all the fields, and then you will have the items for your price list.
+
+Here you can see what the front end could look like.
+
+![Price List](images/Price-List.png) ![Price List](images/Price-List-frontend2.png)
```

Comment:
I think it would be enough with 1 example here, don't you?

Reply:
I think it is nice to have them side by side makes it easier to grasp at least for me ;)